README.md

```diff
@@ -1,31 +1,61 @@
-# NF1_SchwannCell_data_analysis
-Here we analyze [NF1](https://www.sciencedirect.com/science/article/pii/B9780444529022000539) morphology data from the [nf1_cellpainting_data](https://github.com/WayScience/nf1_cellpainting_data) repository.
-Our analysis consists of finding interesting patterns in the NF1 cell morphology data from the [nf1_cellpainting_data](https://github.com/WayScience/NF1_SchwannCell_data_analysis) repository, and then developing, evaluating and interpretting our NF1 model.
-The NF1 model is a binary classifcation model optimized from NF1 cell morphology data to classify cell genotypes {Null, WT} for the NF1 gene.
+# NF1 Schwann cell data analysis
+
+We aim to characterize cell morphology signatures of neurofibromin in Schwann cells. 
+We applied a modified Cell Painting assay on two isogenic Schwann cell lines, one of the wildtype genotype (*NF1+/+*) and one of the null genotype (*NF1-/-*), both from the same patient.
+The modified assay stains for four organelles: nuclei, endoplasmic reticulum, mitochondria, and actin. 
+We applied CellProfiler pipelines to perform quality control, illumination correction, segmentation, and feature extraction. 
+
+![Figure 1](./3.figures/figures/main_figure_1_workflow.png)
+> *Image montage of dataset and analysis workflow.* (A) The workflow of our analysis pipeline demonstrates the steps taken from image analysis to machine learning. (B) Example image montages of the Cell Painting channels and composite image (all channels overlayed) for each NF1 genotype. The scale bar represents 25 μM.
+
+We segmented 22,585 wildtype (WT) and null cells across three plates and utilized 907 significant morphology features representing various organelle shapes and intensity patterns. 
+
+We trained a logistic regression binary classifier to predict the NF1 genotype of single cells. 
+The model has high performance, with training and testing data, with overall accuracy scores of 0.85 and 0.80, respectively.
+
+![Figure 3](./3.figures/figures/main_figure_3_model_eval.png)
+> *Logistic regression model predicts genotype with high performance.* (A) Precision-recall curves comparing the final model applied to shuffled (dashed line) and non-shuffled data (solid line). Applying the model to a shuffled dataset performed worse than the non-shuffled data, demonstrating a biological signal between genotypes. (B) Confusion matrices from the training and testing data splits show higher performance across genotypes in non-shuffled data compared to the shuffled data. (C) Accuracy scores show high performance among both genotypes from the training and testing dataset compared to shuffled data. Both panels B and C use the best-performing hyperparameter model for their plots.
```

Comment:
```suggestion
> *The NF1 model predicts genotype with high performance.* (A) Precision-recall curves comparing the final model applied to shuffled (dashed line) and non-shuffled data (solid line). Applying the model to a shuffled dataset performed worse than the non-shuffled data, demonstrating a biological signal between genotypes. (B) Confusion matrices from the training and testing data splits show higher performance across genotypes in non-shuffled data compared to the shuffled data. (C) Accuracy scores show high performance classifying cells with both genotypes from the training and testing data splits compared to their shuffled counterparts. Both panels B and C use the optimized NF1 model results.
```

Reply:
👍 I will include some of these suggestions, thanks!